steepshot_io/prod_settings.py

```diff
@@ -83,9 +83,9 @@
                       'active_users': 'https://steepshot.org/api/v1/user/active/monthly',
                       'ration_daily': 'https://steepshot.org/api/v1/posts/ratio/daily',
                       'ration_monthly': 'https://steepshot.org/api/v1/posts/ratio/monthly',
-                      'new_users': 'https://qa.steepshot.org/api/v1/user/count/daily',
-                      'new_users_monthly': 'https://qa.steepshot.org/api/v1/user/count/unique',
-                      'users_percent_daily': 'https://qa.steepshot.org/api/v1/user/percent/unique',
+                      'new_users': 'https://steepshot.org/api/v1/user/count/daily',
```

Comment:
Please extract base URL to constants, for example:
``` python
STEEM_V1 = 'https://steepshot.org/api/v1'
STEEM_QA_V1 = 'https://qa.steepshot.org/api/v1'
GOLOS_V1 = 'https://golos.steepshot.org/api/v1'
GOLOS_QA_V1 = 'https://qa.golos.steepshot.org/api/v1'

...
REQUESTS_URL = {
    'new_users': '{url}/user/count/daily',
}
```

And then when you want to call this URL do something like this:
``` python
    res = requests.get(settings.REQUESTS_URL[name_url]).format(url=settings.STEEM_V1).json()  
    # but handle situations when there is no key 'name_url' in settings.REQUESTS_URL
```